### PR TITLE
(Idea) dict style NN outputs and training pipeline

### PR DESCRIPTION
**Pull request by YuriCat**
It would be good update for custom output but some of new implementation are too abstract.

### CODE REVIEW

File: handyrl/evaluation.py [hunk 1]
```diff
@@ -93,7 +95,9 @@ def action(self, env, player, show=False):
 
     def observe(self, env, player, show=False):
         if self.observation:
-            _, v, _, self.hidden = self.planner.inference(env.observation(player), self.hidden)
+            outputs = self.planner.inference(env.observation(player), self.hidden)
```
Comment (ikki407) [inline handyrl/evaluation.py:98]: planner => model??
I think this was changed at previous update.

Comment (YuriCat) [reply handyrl/evaluation.py:98]: Yes, this is a bug. Fixed in #68 and should be fixed in this style too.

Comment (YuriCat) [reply handyrl/evaluation.py]: fixed and remove hidden state from outputs from plan()

**Comment by ikki407** [comment]
LGTM!

**Pull request merged**